### ISSUE TITLE
bug: Revanced Manager thinks that my device is ARMv7

### ISSUE THREAD

**Issue by walrus543** [revanced/revanced-manager#972]
### Type

Error while running the manager

### Bug description

I use Revanced Manager without root (version 1.3.6) on a OnePlus 8 Pro IN2025 - OxygenOS Android 13.

Informations from DevCheck app:
- Hardware: arm64-v8a (64-bit)
- System: Architecture aarch64 (64-bit)

When I want to patch an app (e.g Twitter installed from Google Play), I get this warning:

> Patching on ARMv7 devices is not yet supported and might fail. Proceed anayways?

If I proceed then the process seems to work fine until the end while the manager asks me to install. Installing with the same package name doesn't work of course because of signature inconsistency.

### Steps to reproduce

1. Install Twitter from Google Play
2. Open the manager
3. Select Twitter in the Patcher menu then the patches
4. Hit the `Patch` button

### Android version

Android 13 (OxygenOS)
Android security update: April 5, 2023

### Manager version

v1.3.6

### Target package name

com.twitter.android

### Target package version.

9.94.0-release.0

### Installation type

Non-root

### Patches selected.

Dynamic Color
Hide Ads
Hide Recommanded Users
Remove Screen Capture Restriction
Remove Screenshot Restriction
Spoof Wifi Connection

### Device logs (exported using Manager settings).

```shell
--------- beginning of system
06-30 14:04:21.660 19812 19812 V ZoomWindowDecorView: setLastReportedMergedConfiguration mZoomDisplayHeight: 2376 getDecorView.155862669
06-30 14:04:21.767 19812 19812 I Quality : Skipped: false 3 cost 51.520954 refreshRate 16630803 bit true processName app.revanced.manager.flutter
06-30 14:04:23.719 19812  4626 I Quality : stackInfo :----- pid 19812 at 2023-06-30 14:04:23.714 -----;Cmd line: app.revanced.manager.flutter;"main" prio=5 tid=2 TIMED_WAITING sysTid=19812;  at java.lang.Object.wait(Native Method);  at java.lang.Object.wait(Object.java:442);  at java.lang.Thread.join(Thread.java:1531);  at com.stericson.RootTools.execution.Shell.<init>(Shell.java:85);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495);  at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764);  at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538);  at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51);  at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41);  at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258);  at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295);  at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322);  at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12);  at android.os.Handler.handleCallback(Handler.java:942);  at android.os.Handler.dispatchMessage(Handler.java:99);  at android.os.Looper.loopOnce(Looper.java:240);  at android.os.Looper.loop(Looper.java:351);  at android.app.ActivityThread.main(ActivityThread.java:8364);  at java.lang.reflect.Method.invoke(Native Method);  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584);  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013);
06-30 14:04:25.225 19812  4626 I Quality : stackInfo :----- pid 19812 at 2023-06-30 14:04:25.220 -----;Cmd line: app.revanced.manager.flutter;"main" prio=5 tid=2 TIMED_WAITING sysTid=19812;  at java.lang.Object.wait(Native Method);  at java.lang.Object.wait(Object.java:442);  at java.lang.Thread.join(Thread.java:1531);  at com.stericson.RootTools.execution.Shell.<init>(Shell.java:85);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495);  at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764);  at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538);  at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51);  at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41);  at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258);  at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295);  at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322);  at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12);  at android.os.Handler.handleCallback(Handler.java:942);  at android.os.Handler.dispatchMessage(Handler.java:99);  at android.os.Looper.loopOnce(Looper.java:240);  at android.os.Looper.loop(Looper.java:351);  at android.app.ActivityThread.main(ActivityThread.java:8364);  at java.lang.reflect.Method.invoke(Native Method);  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584);  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013);
06-30 14:04:34.947 29397 29397 D OplusInputMethodUtil: init sDebug to false, init sDebugIme to false, init sAlwaysOn to false
06-30 14:04:34.948 29397 29397 D OplusInputMethodUtil: updateDebugToClass InputMethodManager.DEBUG = false
06-30 14:04:34.948 29397 29397 D OplusInputMethodUtil: updateDebugToClass ImeFocusController.DEBUG = false
06-30 14:04:34.948 29397 29397 D OplusInputMethodUtil: updateDebugToClass BaseInputConnection.DEBUG = false
06-30 14:04:34.948 29397 29397 D OplusInputMethodUtil: updateDebugToClass ImeFocusController.sDebugIme = false
06-30 14:04:34.993 29397 29397 V ZoomWindowDecorView: setLastReportedMergedConfiguration mZoomDisplayHeight: 2376 getDecorView.170919384
06-30 14:04:35.052 29397 29397 I Quality : Skipped: false 4 cost 69.23912 refreshRate 16624866 bit true processName app.revanced.manager.flutter
06-30 14:04:36.956 29397  5893 I Quality : stackInfo :----- pid 29397 at 2023-06-30 14:04:36.951 -----;Cmd line: app.revanced.manager.flutter;"main" prio=5 tid=2 TIMED_WAITING sysTid=29397;  at java.lang.Object.wait(Native Method);  at java.lang.Object.wait(Object.java:442);  at java.lang.Thread.join(Thread.java:1531);  at com.stericson.RootTools.execution.Shell.<init>(Shell.java:85);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513);  at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495);  at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764);  at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538);  at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51);  at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41);  at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258);  at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295);  at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322);  at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12);  at android.os.Handler.handleCallback(Handler.java:942);  at android.os.Handler.dispatchMessage(Handler.java:99);  at android.os.Looper.loopOnce(Looper.java:240);  at android.os.Looper.loop(Looper.java:351);  at android.app.ActivityThread.main(ActivityThread.java:8364);  at java.lang.reflect.Method.invoke(Native Method);  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584);  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013);
06-30 14:04:37.568 29397  5890 I Quality : Blocked msg = Package name: app.revanced.manager.flutter [ schedGroup: 1 schedPolicy: 0 ] process the message: { when=-2s119ms what=0 target=android.os.Handler callback=io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0 } took 2119 ms
06-30 14:04:38.775 29397 29397 D OplusInputMethodManagerInternal: get inputMethodManager extension: com.android.internal.view.IInputMethodManager$Stub$Proxy@7cffb4d
06-30 14:06:36.430 29397 29397 I Quality : Skipped: false 1 cost 24.582527 refreshRate 16612856 bit true processName app.revanced.manager.flutter
06-30 14:08:43.816 29397 29397 I Quality : Skipped: false 1 cost 21.304333 refreshRate 16629408 bit true processName app.revanced.manager.flutter
06-30 14:11:31.756 29397 29397 V ZoomWindowDecorView: removeZoomView
06-30 14:11:54.099 29397 29397 I Quality : Skipped: false 2 cost 38.783764 refreshRate 16625016 bit true processName app.revanced.manager.flutter
06-30 14:11:54.105 29397 29397 V ZoomWindowDecorView: setLastReportedMergedConfiguration mZoomDisplayHeight: 2376 getDecorView.188499678
06-30 14:12:24.373 29397 29397 V ZoomWindowDecorView: removeZoomView
06-30 14:12:26.967 29397 29397 D ActivityThread: do gfx trim 40 success
--------- beginning of main
06-30 14:13:41.556 29397 29397 D AutofillManager: Fill dialog is enabled:false, hints=[password, passwordAuto, creditCardNumber, creditCardSecurityCode, creditCardExpirationDate]
06-30 14:13:41.576 29397 29397 D WindowManager: Add to mViews: DecorView@c0d4ef0[MainActivity],pkg= app.revanced.manager.flutter
06-30 14:13:41.585 29397 29397 D InputEventReceiver: Input log is disabled
06-30 14:13:41.597 29397 29397 I SurfaceView: 53984683 onAttachedToWindow
06-30 14:13:41.610 29397 29397 D BufferQueueConsumer: [](id:72d500000004,api:0,p:-1,c:29397) connect: controlledByApp=false
06-30 14:13:41.610 29397 29397 E IPCThreadState: attemptIncStrongHandle(83): Not supported
06-30 14:13:41.610 29397 29397 E IPCThreadState: attemptIncStrongHandle(73): Not supported
06-30 14:13:41.610 29397 29397 E IPCThreadState: attemptIncStrongHandle(64): Not supported
06-30 14:13:41.610 29397 29397 V ZoomWindowDecorView: setLastReportedMergedConfiguration mZoomDisplayHeight: 2376 getDecorView.202198768
06-30 14:13:41.611 29397  5895 D BufferQueueProducer: [VRI[MainActivity]#4(BLAST Consumer)4](id:72d500000004,api:1,p:29397,c:29397) connect: api=1 producerControlledByApp=true
06-30 14:13:41.614 29397 29397 D BufferQueueConsumer: [](id:72d500000005,api:0,p:-1,c:29397) connect: controlledByApp=false
06-30 14:13:41.614 29397 29397 I SurfaceView: 53984683 visibleChanged -- surfaceCreated 
06-30 14:13:41.615 29397  9140 D BufferQueueProducer: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#5(BLAST Consumer)5](id:72d500000005,api:1,p:29397,c:29397) connect: api=1 producerControlledByApp=true
06-30 14:13:41.617 29397 29397 I SurfaceView: 53984683 surfaceChanged -- format=4 w=1080 h=2376
06-30 14:13:41.673 29397 29397 I Quality : Skipped: false 3 cost 50.076145 refreshRate 16628601 bit true processName app.revanced.manager.flutter
06-30 14:13:41.877 29397  9137 I flutter : dynamic_color: Core palette detected.
06-30 14:13:41.923 29397 29397 E SurfaceSyncer: Failed to find sync for id=0
06-30 14:13:41.923 29397 29397 D VRI[MainActivity]: registerCallbacksForSync syncBuffer=false
06-30 14:13:41.926 29397  5895 D VRI[MainActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true syncBuffer=false
06-30 14:13:41.927 29397  5895 W Parcel  : Expecting binder but got null!
06-30 14:13:41.927 29397 29397 D VRI[MainActivity]: draw finished.
06-30 14:13:41.959 29397 29397 D VRI[MainActivity]: onFocusEvent true
06-30 14:13:42.115 29397 29397 W System.err: com.stericson.RootTools.exceptions.RootDeniedException: Root Access Denied
06-30 14:13:42.115 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.<init>(Shell.java:113)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41)
06-30 14:13:42.115 29397 29397 W System.err: 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258)
06-30 14:13:42.115 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
06-30 14:13:42.115 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322)
06-30 14:13:42.115 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
06-30 14:13:42.115 29397 29397 W System.err: 	at android.os.Handler.handleCallback(Handler.java:942)
06-30 14:13:42.115 29397 29397 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
06-30 14:13:42.115 29397 29397 W System.err: 	at android.os.Looper.loopOnce(Looper.java:240)
06-30 14:13:42.115 29397 29397 W System.err: 	at android.os.Looper.loop(Looper.java:351)
06-30 14:13:42.115 29397 29397 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:8364)
06-30 14:13:42.115 29397 29397 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
06-30 14:13:42.115 29397 29397 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
06-30 14:13:43.216 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:13:44.174 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:13:44.324 29397 29397 D UxIconPackageManagerExt: loadItemIcon is support edit icon: false
06-30 14:13:44.326 29397 29397 D UxIconPackageManagerExt: UxIcon checkChooseIconsRootPath false
06-30 14:13:44.360 29397 29397 D UxIconPackageManagerExt: loadItemIcon is support edit icon: false
06-30 14:13:44.360 29397 29397 D UxIconPackageManagerExt: UxIcon checkChooseIconsRootPath false
06-30 14:13:44.396 29397 29397 D UxIconPackageManagerExt: loadItemIcon is support edit icon: false
06-30 14:13:44.396 29397 29397 D UxIconPackageManagerExt: UxIcon checkChooseIconsRootPath false
06-30 14:13:44.415 29397 29397 D UxIconPackageManagerExt: loadItemIcon is support edit icon: false
06-30 14:13:44.415 29397 29397 D UxIconPackageManagerExt: UxIcon checkChooseIconsRootPath false
06-30 14:13:45.188 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:13:45.252 29397 29397 D UxIconPackageManagerExt: loadItemIcon is support edit icon: false
06-30 14:13:45.252 29397 29397 D UxIconPackageManagerExt: UxIcon checkChooseIconsRootPath false
06-30 14:13:47.691 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:13:58.124 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:06.760 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:34.423 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:34.425 29397 29397 D OplusSystemUINavigationGesture: [GESTURE_BUTTON] swipe from 1
06-30 14:14:34.425 29397 29397 D OplusSystemUINavigationGesture: [GESTURE_BUTTON] Hit Gesture Region !
06-30 14:14:34.910 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:34.911 29397 29397 D OplusSystemUINavigationGesture: [GESTURE_BUTTON] swipe from 1
06-30 14:14:34.911 29397 29397 D OplusSystemUINavigationGesture: [GESTURE_BUTTON] Hit Gesture Region !
06-30 14:14:34.999 29397 29397 D VRI[MainActivity]: onFocusEvent false
06-30 14:14:35.249 29397  8865 D VRI[MainActivity]: dispatchAppVisibility visible:false
06-30 14:14:35.265 29397 29397 I SurfaceView: 53984683 surfaceDestroyed
06-30 14:14:35.271 29397  9140 D BufferQueueProducer: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#5(BLAST Consumer)5](id:72d500000005,api:1,p:29397,c:29397) disconnect: api 1
06-30 14:14:35.320 29397 29397 D BufferQueueProducer: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#5(BLAST Consumer)5](id:72d500000005,api:0,p:-1,c:29397) disconnect: api -1
06-30 14:14:35.320 29397 29397 D BufferQueueConsumer: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#5(BLAST Consumer)5](id:72d500000005,api:0,p:-1,c:29397) disconnect
06-30 14:14:35.322 29397  5895 D SurfaceView: 195832879positionLost mSurfaceControl is null return;
06-30 14:14:35.322 29397  5895 D BufferQueueProducer: [VRI[MainActivity]#4(BLAST Consumer)4](id:72d500000004,api:1,p:29397,c:29397) disconnect: api 1
06-30 14:14:35.325 29397 29397 D BufferQueueConsumer: [VRI[MainActivity]#4(BLAST Consumer)4](id:72d500000004,api:0,p:-1,c:29397) disconnect
06-30 14:14:35.328 29397 29397 D VRI[MainActivity]: setWindowStopped stopped:true
06-30 14:14:35.494 29397  9141 E libEGL  : call to OpenGL ES API with no current context (logged once per thread)
06-30 14:14:35.500 29397 29397 D OplusScrollToTopManager: unregisterGuidePopupDismissReceiverInSystemUI
06-30 14:14:35.500 29397 29397 D OplusScrollToTopManager: Receiver not registered: android.view.OplusScrollToTopManager$1@4f9c26c
06-30 14:14:35.500 29397 29397 D OplusScrollToTopManager: unregisterSystemUIBroadcastReceiver 
06-30 14:14:35.500 29397 29397 D OplusScrollToTopManager: java.lang.IllegalArgumentException: Receiver not registered: android.view.OplusScrollToTopManager$2@31ce035
06-30 14:14:35.502 29397 29397 V ZoomWindowDecorView: removeZoomView
06-30 14:14:36.176 29397 29397 D AutofillManager: Fill dialog is enabled:false, hints=[password, passwordAuto, creditCardNumber, creditCardSecurityCode, creditCardExpirationDate]
06-30 14:14:36.186 29397 29397 D WindowManager: Add to mViews: DecorView@3b5650e[MainActivity],pkg= app.revanced.manager.flutter
06-30 14:14:36.197 29397 29397 D InputEventReceiver: Input log is disabled
06-30 14:14:36.201 29397 29397 I SurfaceView: 115467841 onAttachedToWindow
06-30 14:14:36.208 29397 29397 D BufferQueueConsumer: [](id:72d500000006,api:0,p:-1,c:29397) connect: controlledByApp=false
06-30 14:14:36.208 29397 29397 E IPCThreadState: attemptIncStrongHandle(65): Not supported
06-30 14:14:36.209 29397 29397 V ZoomWindowDecorView: setLastReportedMergedConfiguration mZoomDisplayHeight: 2376 getDecorView.62219534
06-30 14:14:36.209 29397  5895 D BufferQueueProducer: [VRI[MainActivity]#6(BLAST Consumer)6](id:72d500000006,api:1,p:29397,c:29397) connect: api=1 producerControlledByApp=true
06-30 14:14:36.214 29397 29397 D BufferQueueConsumer: [](id:72d500000007,api:0,p:-1,c:29397) connect: controlledByApp=false
06-30 14:14:36.215 29397 29397 I SurfaceView: 115467841 visibleChanged -- surfaceCreated 
06-30 14:14:36.215 29397  9565 D BufferQueueProducer: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#7(BLAST Consumer)7](id:72d500000007,api:1,p:29397,c:29397) connect: api=1 producerControlledByApp=true
06-30 14:14:36.217 29397 29397 I SurfaceView: 115467841 surfaceChanged -- format=4 w=1080 h=2376
06-30 14:14:36.263 29397 29397 I Quality : Skipped: false 3 cost 51.908882 refreshRate 16628754 bit true processName app.revanced.manager.flutter
06-30 14:14:36.564 29397  9564 I flutter : dynamic_color: Core palette detected.
06-30 14:14:36.594 29397 29397 E SurfaceSyncer: Failed to find sync for id=0
06-30 14:14:36.594 29397 29397 D VRI[MainActivity]: registerCallbacksForSync syncBuffer=false
06-30 14:14:36.596 29397  5895 D VRI[MainActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true syncBuffer=false
06-30 14:14:36.596 29397  5895 W Parcel  : Expecting binder but got null!
06-30 14:14:36.596 29397 29397 D VRI[MainActivity]: draw finished.
06-30 14:14:36.613 29397 29397 D VRI[MainActivity]: onFocusEvent true
06-30 14:14:36.724 29397 29397 W System.err: com.stericson.RootTools.exceptions.RootDeniedException: Root Access Denied
06-30 14:14:36.724 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.<init>(Shell.java:113)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:513)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.stericson.RootTools.execution.Shell.startRootShell(Shell.java:495)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.stericson.RootTools.internal.RootToolsInternalMethods.isAccessGiven(RootToolsInternalMethods.java:764)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.stericson.RootTools.RootTools.isAccessGiven(RootTools.java:538)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.gokul.root.RootPlugin.isRooted(RootPlugin.java:51)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.gokul.root.RootPlugin.onMethodCall(RootPlugin.java:41)
06-30 14:14:36.724 29397 29397 W System.err: 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258)
06-30 14:14:36.724 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
06-30 14:14:36.724 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:322)
06-30 14:14:36.724 29397 29397 W System.err: 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
06-30 14:14:36.724 29397 29397 W System.err: 	at android.os.Handler.handleCallback(Handler.java:942)
06-30 14:14:36.724 29397 29397 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
06-30 14:14:36.724 29397 29397 W System.err: 	at android.os.Looper.loopOnce(Looper.java:240)
06-30 14:14:36.724 29397 29397 W System.err: 	at android.os.Looper.loop(Looper.java:351)
06-30 14:14:36.724 29397 29397 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:8364)
06-30 14:14:36.724 29397 29397 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
06-30 14:14:36.724 29397 29397 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
06-30 14:14:37.297 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:39.407 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:42.554 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:44.106 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
06-30 14:14:46.026 29397 29397 V AutofillManager: requestHideFillUi(null): anchor = null
```

### Screenshots or videos

![photo_2023-06-30_14-16-24](https://github.com/revanced/revanced-manager/assets/5938828/b093e90c-fab2-44be-8362-977509fa676f)

### Solution

No problem when using the apk version from APKMirror.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager and not related to patching errors or patches

**Comment by TheAabedKhan** [comment]
Can you also attach a screenshot of the `About` section located in the `Settings` page?

**Comment by walrus543** [comment]
Sure.
![photo_2023-06-30_14-54-58](https://github.com/revanced/revanced-manager/assets/5938828/037adfe6-3eb9-4569-80f6-c40bbfa8c0e8)

**Comment by TheAabedKhan** [comment]
After reviewing the code, I can confirm that you got the warning message because you are trying to patch a `split apk` which is not supported. However, the dialog should mention the exact issue which it doesn't. This issue will be fixed in the next update. 

Thank you so much for taking your valuable time to report the issue.

**Comment by Duvio** [comment]
> After reviewing the code, I can confirm that you got the warning message because you are trying to patch a `split apk` which is not supported. However, the dialog should mention the exact issue which it doesn't. This issue will be fixed in the next update.

YouTube is installed as a split apk, right? At least i cannot find any apks from reputable sources like apkmirror that are NOT split.

**Comment by RaWolFoX** [comment]
> > After reviewing the code, I can confirm that you got the warning message because you are trying to patch a `split apk` which is not supported. However, the dialog should mention the exact issue which it doesn't. This issue will be fixed in the next update.
> 
> YouTube is installed as a split apk, right? At least i cannot find any apks from reputable sources like apkmirror that are NOT split. I'm asking because i have a separate issue that could be related to this.

You can get single apk from apkmirror. Just select universal apk from download list.

**Comment by Duvio** [comment]
> > > After reviewing the code, I can confirm that you got the warning message because you are trying to patch a `split apk` which is not supported. However, the dialog should mention the exact issue which it doesn't. This issue will be fixed in the next update.
> > 
> > 
> > YouTube is installed as a split apk, right? At least i cannot find any apks from reputable sources like apkmirror that are NOT split. I'm asking because i have a separate issue that could be related to this.
> 
> You can get single apk from apkmirror. Just select universal apk from download list.

Awesome. This also, for some reason, solves #908, which has been locked because of spam (?).

**Comment by EvadeMaster** [comment]
🎉 Closing this issue as it been fixed already long ago by #973 

Feels free to reopen if this doesn't solve your issue.
